pandas/tests/arrays/test_timedeltas.py

```diff
@@ -160,11 +172,87 @@ def test_min_max(self):
         result = arr.max(skipna=False)
         assert result is pd.NaT
 
-    @pytest.mark.parametrize("skipna", [True, False])
-    def test_min_max_empty(self, skipna):
-        arr = TimedeltaArray._from_sequence([])
-        result = arr.min(skipna=skipna)
+    def test_sum(self):
+        tdi = pd.TimedeltaIndex(["3H", "3H", "NaT", "2H", "5H", "4H"])
+        arr = tdi.array
+
+        result = arr.sum(skipna=True)
+        expected = pd.Timedelta(hours=17)
+        assert isinstance(result, pd.Timedelta)
+        assert result == expected
+
+        result = tdi.sum(skipna=True)
+        assert isinstance(result, pd.Timedelta)
+        assert result == expected
+
+        result = arr.sum(skipna=False)
+        assert result is pd.NaT
+
+        result = tdi.sum(skipna=False)
+        assert result is pd.NaT
+
+        result = arr.sum(min_count=9)
+        assert result is pd.NaT
+
+        result = tdi.sum(min_count=9)
+        assert result is pd.NaT
+
+        result = arr.sum(min_count=1)
+        assert isinstance(result, pd.Timedelta)
+        assert result == expected
+
+        result = tdi.sum(min_count=1)
```

Comment:
The `min_count` was mainly added for the empty case, and for this it does not seem to work:

```
In [27]: tdi = pd.TimedeltaIndex([])                                                                                                                                                                               

In [28]: tdi.sum()                                                                                                                                                                                                 
Out[28]: NaT

In [29]: tdi.sum(min_count=1)                                                                                                                                                                                      
Out[29]: NaT

In [30]: tdi.sum(min_count=0)                                                                                                                                                                                      
Out[30]: NaT
```

If we follow the same behaviour as for numerical sum, the result should be 0 instead of NaT

Reply:
I think you're right